controllers/comments.controller.js

```diff
@@ -16,4 +15,19 @@ const getCommentsByArticleId = (req, res, next) => {
     });
 };
 
-module.exports = { getCommentsByArticleId };
+const addCommentByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+
+
+  insertCommentByArticleId(article_id, username, body)
+    .then((comment) => {
+      res.status(201).json({ comment });
```

Comment:
why are we using res.json here ?

Reply:
code would not pass without it says the data must be parsed